packages/editor/src/plugins/dropzone-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+
+import { DropzoneImageEditor } from './components/editor/editor'
+import { BackgroundShape, DropzoneVisibility } from './types'
+import {
+  type EditorPlugin,
+  type EditorPluginProps,
+  object,
+  string,
+  child,
+  number,
+  list,
+  optional,
+} from '../../plugin'
+
+export const positionData = object({
+  top: number(0),
+  left: number(0),
+})
+
+export const layoutData = object({
+  width: number(200),
+  height: number(70),
+})
```

Comment:
How are those numbers defined currently? Size of original image? What happens on smaller displays when the image resizes...

Reply:
Good point, will try to make a change so the static view looks acceptable on mobile

Reply:
> Good point, will try to make a change so the static view looks acceptable on mobile

Nah, IMHO a good user experience on mobile for the static view is essential before merging this PR...

Reply:
Thats what I wrote, I will make a change to make the mobile look on static acceptable. Mostly finished this change, just need to fix one more bug. Will push before 12

Reply:
Pushed a fix and the static view works in mobile now. I think we would need to get this on staging to test more properly with mobile devices

Reply:
Going to mark this as resolved